README.md

```diff
@@ -1,16 +1,16 @@
-### Testing React apps with Nightwatch - javascript tool for data driven testing
+### Testing React apps with Nightwatch - Data Driven Testing @ Syncano
 
 This is the third part of End to End testing of React apps with Nightwatch series.
 In the [previous posts](https://www.syncano.io/blog/testing-syncano/) We've talked about Nightwatch:
 
 - Using `before()` and `after()` hooks in your tests
 - Extending Nightwatch with custom commands
 
-In this part I'll focus on script that let us feed dashboard with data, and then use it in example e2e test.
+In this part I'll focus on a script that let us feed dashboard with data, and then use it in example e2e test.
```

Comment:
I'd write here about the problems we faced and why we decided to go with this approach. A bit of a background.